components/developer/libtool/Makefile

```diff
@@ -52,12 +55,13 @@ include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/configure.mk
 include $(WS_MAKE_RULES)/ips.mk
 
+PATH=$(PATH.gnu)
+
 CONFIGURE_OPTIONS  +=		--infodir=$(CONFIGURE_INFODIR)
 CONFIGURE_OPTIONS  +=		--disable-static
 CONFIGURE_OPTIONS  +=		PERL="$(PERL)"
 
-
-COMPONENT_PREP_ACTION = (cd $(@D) ; ./bootstrap)
+COMPONENT_PREP_ACTION = (cd $(@D) ; ./bootstrap --force)
```

Comment:
prep step fails if you avoid touching  $(SOURCE_DIR)/README-release  (look at solaris-userland)